docs/VnVPlan/VnVPlan.tex

```diff
@@ -761,15 +938,15 @@ \subsection{Traceability Between Test Cases and Requirements}
   \hline
   \textbf{Req. ID} & \textbf{Test ID's} \\
   \hline
-  FR1 & \\ \hline
+  FR1 & TC-FR1, TC-FR2\\ \hline
   FR2 & \\ \hline
```

Comment:
could you remove this from the SRS by doing a strikeout like Rafeed did here https://github.com/Spitgranger/SyncMaster/pull/221